docs/core/tools/dotnet-package-search.md

```diff
@@ -0,0 +1,111 @@
+---
+title: dotnet package search command
+description: Searches a given source using the query string provided. If no sources are specified, all sources defined in the NuGet.Config file are used
+author: Nigusu-Allehu
+ms.date: 10/26/2023
+---
+# dotnet package search
+
+**This article applies to:** ✔️ .NET 8.0.2xx SDK and later versions
+
+## Name
+
+`dotnet package search` - Searches for a NuGet package.
+
+## Synopsis
+
+```dotnetcli
+dotnet package search [search terms] [options]
+    [--source <SOURCE>]
+    [--exact-match]
+    [--prerelease]
+    [--interactive]
+    [--take <TAKE>]
+    [--skip <SKIP>]
+
+dotnet package search -h|--help
+```
+
+## Description
+
+The `dotnet package search` command searches for a NuGet package.
+
+## Arguments
+
+- **`search terms`**
+
+  Specifies the search term to filter results. Use this argument to search for packages matching the provided query. Example: `dotnet package search json`.
+
+## Options
+
+- **`--source <SOURCE>`**
+
+  The package source to search. You can pass multiple --source options to search multiple package sources.
+
+- **`--exact-match`**
+
+  Return exact matches only as a search result.
+
+- **`--prerelease`**
+
+    Allow prerelease packages to be shown.
+
+- **`--interactive`**
+
+    Allows the command to stop and wait for user input or action (for example to complete authentication).
+
+- **`--take`**
+
+    The number of results to return. The default value is 20.
+
+- **`--skip`**
+
+    The number of results to skip, for pagination. The default value is 0.
+
+[!INCLUDE [help](../../../includes/cli-help.md)]
+
+## Examples
```

Comment:
See [dotnet tool search](https://learn.microsoft.com/en-us/dotnet/core/tools/dotnet-tool-search) for an example of how to format these examples and how to write the text that introduces each one. Also, note that the opening triple backtick for each output example can specify output as the type:
~~~
```output
~~~